mitosheet/src/mito/utils/actions.tsx

```diff
@@ -1649,6 +1649,106 @@ export const getActions = (
             searchTerms: ['docs', 'documentation', 'help', 'support'],
             tooltip: "Documentation, tutorials, and how-tos on all functionality in Mito."
         },
+        [ActionEnum.Select_Columns]: {
+            type: 'build-time',
+            staticType: ActionEnum.Select_Columns,
+            longTitle: 'Select column',
+            actionFunction: () => {
+                // We turn off editing mode, if it is on
+                setEditorState(undefined);
+
+                // We close the editing taskpane if its open
+                closeOpenEditingPopups();
+
+                // We open the column selection taskpane
+                let newStartingColumnIndex = gridState.selections[0].startingColumnIndex;
+                let newEndingColumnIndex = gridState.selections[0].endingColumnIndex;
+                for (const selection of gridState.selections) {
+                    newStartingColumnIndex = Math.min(newStartingColumnIndex, selection.startingColumnIndex);
+                    newEndingColumnIndex = Math.max(newEndingColumnIndex, selection.endingColumnIndex);
+                }
```

Comment:
So this works differently than Excel on multiple selections. If you select multiple ranges in excels and then press control + space, it only selects the columns from the first selection. 

Would be down to match this behavior -- so pretty much just looking at `gridState.selections[0]`.